TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsChildTransmissionTargetTableViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
Please use regular copyright format that we have.

Reply:
damn, sorry!